### PR TITLE
added and resized logo to work responsively with page layout.

### PR DESCRIPTION
**Pull request by CharlieStackup**
Added logo to page.

added width and padding to logo to fit requirements.

Made the logo responsive to the page layout using a mobile first approach.

### CODE REVIEW

**Comment by jamie-endeavour** [comment]
Task linked: [CU-9evtjz Add New Logo](https://app.clickup.com/t/9evtjz)

File: web/_includes/partials/_hero.njk [hunk 1]
```diff
@@ -2,6 +2,7 @@
   <div class="max-w-screen-xl mx-auto">
     <div class="relative z-10 pb-8 sm:pb-8 md:pb-20 lg:max-w-2xl lg:w-full lg:pb-28 xl:pb-32">
       <main class="mt-10 mx-auto max-w-screen-xl px-4 sm:mt-12 sm:px-6 md:mt-16 lg:mt-20 lg:px-8 xl:mt-28">
+      <img src="/assets/stackup-red.svg" alt="Logo should appear here" class="logo-img pb-8 md:pb-5 md:mx-auto lg:mx-0">
```
Comment (jamie-endeavour) [inline web/_includes/partials/_hero.njk:5]: Hey @CharlieStackup to fix the bug we found where the logo pings off to the left on smaller devices you need to add the following class `sm:mx-auto` therefore the img tag will need to look like:

```html
<img src="/assets/stackup-red.svg" alt="Logo should appear here" class="logo-img pb-8 md:pb-5 sm:mx-auto md:mx-auto lg:mx-0">
```

**Review by jamie-endeavour — CHANGES_REQUESTED** [review]
Just needs that quick class name change for me. Once @glenwheelerstackup is happy with the look of it then we can ship it! 🚢

**Comment by glenwheelerstackup** [comment]
Looks good dude, looks to have worked!

**Pull request merged**